docs/reference/indices/index-mgmt.asciidoc

```diff
@@ -236,3 +215,31 @@ earlier.
 GET /my-index-000001,my-index-000002
 --------------------------------------------------
 // TEST[continued]
+
+[float]
+[[manage-enrich-policies]]
+=== Manage enrich policies
+
+Use the *Enrich Policies* view to add data from your existing indices to incoming documents during ingest.
+An enrich policy contains:
+
+* The policy type that determines how the processor matches the enrich data to incoming documents
+* The source indices that store enrich data as documents
+* The fields from the source indices used to match incoming documents
+* The enrich fields containing enrich data from the source indices that you want to add to incoming documents
+
+[role="screenshot"]
+image::images/index-mgmt/management-enrich-policies.png[Enrich policies]
+
+
+When creating an enrich policy, the UI walks you through the configuration setup and selecting the fields.
+Before you can use the policy with an enrich processor, you must execute the policy.
```

Comment:
Should we mention ES|QL here as well? cc @stratoula

Reply:
Doesn't hurt to mention it here too. Maybe something like:

```suggestion
Before you can use the policy with an enrich processor or {esql} query, you must execute the policy.
```

Reply:
I like it!!